R/mlr_reflections.R

```diff
@@ -74,9 +74,9 @@ local({
   ### Task
   # task types + constructors
   mlr_reflections$task_types = rowwise_table(.key = "type",
-    ~type,     ~package, ~task,         ~learner,         ~prediction,         ~measure,
-    "regr",    "mlr3",   "TaskRegr",    "LearnerRegr",    "PredictionRegr",    "MeasureRegr",
-    "classif", "mlr3",   "TaskClassif", "LearnerClassif", "PredictionClassif", "MeasureClassif"
+    ~type, ~package, ~task, ~learner, ~prediction, ~measure,
+    "regr", "mlr3", "TaskRegr", "LearnerRegr", "PredictionRegr", "MeasureRegr",
+    "classif", "mlr3", "TaskClassif", "LearnerClassif", "PredictionClassif", "MeasureClassif"
```

Comment:
@lorenzwalthert DO you have an idea why this alignment is not honoured?

Reply:
Because left alignment only works with `=`, not with `~`. See https://github.com/lorenzwalthert/mlr3/pull/2#discussion_r619304282

Reply:
I tried for a few hours and I did not get to work it easily... Maybe I can look into it again since we are on it.

Reply:
This is really niche so really think about if you want to do this :)

Would be great and appreciated of course though 👍

Reply:
Ok, I found a way to do it and merged it into master: https://github.com/r-lib/styler/pull/785. After the other alignment things were implemented properly, it seemed much easier than I thought initially. Are you happy now? xD

Reply:
Thanks! Just restyled but it seems it not yet working 🙈

Reply:
Because you have an extra wurst with `.key = '...'` -.-. Fix on the way...

Reply:
Please try again xD

Reply:
This seems resolved with latest styling.